src/battle_util.c

```diff
@@ -6418,7 +6375,10 @@ void BattleScriptPushCursorAndCallback(const u8 *BS_ptr)
 {
     BattleScriptPushCursor();
```

Comment:
Couldn't BattleScriptCall be used here as well?

Reply:
Definitely can! Done :)